app/controllers/forms_controller.rb

```diff
@@ -264,4 +263,8 @@ def prepare_and_render_form
     def load_form
       @form = Form.find(params[:id])
     end
+
+    def form_params
+      params.require(:form).permit(:name, :smsable, :allow_incomplete, :access_level)
```

Comment:
Pretty sure there is another param that needs to be allowed here. Can't remember name but there is a big mutliselect box for which users get access to api.